### DISCUSSION TITLE
Need help setting up vscode-neovim in windows

### DISCUSSION THREAD

**Discussion by RiteshChepuri** [vscode-neovim/vscode-neovim#2204] [Q&A]
I took a look at various public repositories settings.json but I couldn't find settings for windows. I'm trying to setup vscode-neovim in windows, I have installed neovim in windows 11 using scoop and the path for the neovim is 
C:\Users\username\scoop\apps\neovim\current\bin\nvim.exe and my path for neovim init.lua C:\Users\ritesh\AppData\Local\nvim/init.lua. 
but when I try to include these paths in my settings.json I have been getting error. Need help

**Comment by justinmk** [comment]
> but when I try to include these paths in my settings.json I have been getting error

don't describe. say *exactly* which setting you set, and exactly which error was given. show a screenshot too.

also [paste the logs](https://github.com/vscode-neovim/vscode-neovim?tab=readme-ov-file#troubleshooting) here.
